Examples/Scripts/Python/particle_gun.py

```diff
@@ -97,47 +79,36 @@ def addParticleGun(
         random number generator
     """
```

Comment:
- [ ] Missing `phiConfig` description in docstring
- [ ] Missing `multiplicity` description in docstring